Ginger/GingerCore/Drivers/JavaDriverLib/JavaDriver.cs

```diff
@@ -1910,6 +1977,8 @@ public static ElementInfo GetControlInfoFromPayLoad(PayLoad pl)
             JEI.Value = pl.GetValueString();
             JEI.Path = pl.GetValueString();
             JEI.XPath = pl.GetValueString();
+            JEI.ElementTypeEnum = JavaPlatform.GetElementType(JEI.ElementType);
+            JEI.IsAutoLearned = true;
```

Comment:
This method is called from many places and isAutolearned can not be set to true by default